documentation/docs/20-core-concepts/40-page-options.md

```diff
@@ -62,6 +62,8 @@ Note that you can still prerender pages that load data based on the page's param
 
 Accessing [`url.searchParams`](/docs/load#using-url-data-url) during prerendering is forbidden. If you need to use it, ensure you are only doing so in the browser (for example in `onMount`).
 
+Pages with [actions](/docs/form-actions) cannot be prerendered, because a server must be able to handle the action `POST` requests.
```

Comment:
Is this enough explanation? People could go "yeah so why can't it handle that?" - then again, I'm not sure how to explain this without explaining a thousand other things.

Reply:
if people come back with further questions we can try and answer them, but i think the important thing is to clarify 'you can't do this' rather than get into the weeds of _why_ you can't do this